google/resource_compute_instance_template.go

```diff
@@ -529,10 +523,10 @@ func resourceComputeInstanceTemplateCreate(d *schema.ResourceData, meta interfac
 		}
 		_scheduling := _schedulings[0].(map[string]interface{})
 
-		if vp, okp := _scheduling["automatic_restart"]; okp {
-			instanceProperties.Scheduling.AutomaticRestart = googleapi.Bool(vp.(bool))
-			forceSendFieldsScheduling = append(forceSendFieldsScheduling, "AutomaticRestart")
-		}
+		// "automatic_restart" has a default value and is always safe to dereference
+		automaticRestart := _scheduling["automatic_restart"].(bool)
+		instanceProperties.Scheduling.AutomaticRestart = googleapi.Bool(automaticRestart)
+		forceSendFieldsScheduling = append(forceSendFieldsScheduling, "AutomaticRestart")
```

Comment:
I don't think you do in this case, as the boolean is of type `*bool`. I can remove it in a subsequent PR